### PR TITLE
New Slack method

### PR DESCRIPTION
**Pull request by lrochette**
Allow to post to a channel using a channel name and token instead of webhook URL

### CODE REVIEW

File: workflows/slack/versions/0.0.2/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Slack
+
+## Summary
+
+A template that enables you to send messages to a Slack channel using a webhook URL.
+
+## Templates
+
+1. [send-message](https://github.com/codefresh-io/argo-hub/blob/main/workflows/slack/versions/0.0.2/docs/send-message.md) 
+
```
Comment (eti-codefresh) [inline workflows/slack/versions/0.0.2/README.md:10]: need to add post-to-channel here

File: workflows/slack/versions/0.0.2/docs/send-message.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  generateName: slack-send-message-
+spec:
+  entrypoint: main
+  templates:
+  - name: main
+    dag:
+      tasks:
+      - name: send-message
+        templateRef:
+          name: argo-hub.slack.0.0.1
```
Comment (eti-codefresh) [inline workflows/slack/versions/0.0.2/docs/send-message.md:31]: need to change to 0.0.2

Comment (lrochette) [reply workflows/slack/versions/0.0.2/docs/send-message.md]: Done

File: workflows/slack/versions/0.0.2/README.md (hunk 1 above)
Comment (lrochette) [reply workflows/slack/versions/0.0.2/README.md:11]: Added with relative path

**Review by eti-codefresh — APPROVED** [review]

**Pull request closed without merging**